app/views/posts/show.html.erb

```diff
@@ -17,4 +17,7 @@
     </div>
     </div>
 </div>
-
+<br>
+<%= form_with(model: @new_like, url: user_post_likes_path(@user, @post)) do |form| %>
+  <%= form.submit "Like" %>
+<% end %>
```

Comment:
:red_circle: Kindly make sure your like feature is working correctly. Nothing happens when I click the like button.

![Peek 2023-09-01 10-45](https://github.com/husush23/Blog-App/assets/26736582/ad817b3c-04ce-4a51-83a8-45a40db80313)